daily_snapshot_terraform/modules/daily_snapshot/service/mainnet_cron_job

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+cd "$BASE_FOLDER" || exit
```

Comment:
it's not an error?

Reply:
Without an error code, `exit` uses the error status of the last executed command.

Reply:
I didn't either! shellcheck taught me. :D